### ISSUE TITLE
note creation date disappears after update

### ISSUE THREAD

**Issue by ujibang** [SoftInstigate/restheart-notes-example#1]
After creation:

```
"date": {
    "$date": 1447396591487
  }
```

After update it gets replaced by

```
"date": 1447320330638,
```

leading to wrong _created on_ field in the UI

**Comment by ujibang** [comment]
fixed in commit e1ba03b08adb4887bcbe67e481b7fb48537acbf6

**Comment by ujibang** [comment]
fixed in commit e1ba03b
